source/paper/Template.tex

```diff
@@ -41,15 +46,15 @@ \section{Some Section}
 
 \newpage{}
 
-\input{output/tables/top_gdp}
+\input{top_gdp}
 
 \newpage{}
 
-\input{source/figures/gdp_educ}
+\input{gdp_educ}
 
 \newpage{}
 \appendix
 \bibliographystyle{apalike}
-\bibliography{source/paper/References}
+\bibliography{References}
```

Comment:
Previously we had `/source/paper/References`. If we point only to `References` I think we will need `/source/paper` on the input path. I've given that a try above. Can you see if it allows us to compile the bibliography via `scons`?

Reply:
To do:
* Look at where `scons` is putting the BBL file
* See what happens if we move references to `/source/refs/References.bib` but keep the line above as `\bibliography{References}`.

Reply:
@jmshapir I tried some of these suggestions but haven't gotten it to work so far. There was one way I did get it to work which I pushed to a new branch, https://github.com/JMSLab/Template/tree/85-testing. I actually prefer this new way but if you don't like it, I will take another shot at the suggestions.

Reply:
@ew487 that is a great idea! 🤯 

I didn't know about the `ifthen` package but it [looks like](https://ctan.org/pkg/ifthen?lang=en) it ships with LaTeX which is nice.

I'm going to close this pull without merge and open a new one for the alternative you've proposed.